HISTORY.rst

```diff
@@ -1,6 +1,45 @@
 =======
 History
 =======
+3.41.0 (2012-08-04)
+-------------------
+
+Changes
+
+* CCI has a new Metadata ETL task that allows adding fields to existing layouts. Sections can be selected using an index (#2766)
+
+* We added a task to enable an Einstein prediction (thanks, @erikperkins!) (#2778)
+
+* We added standard flows for releasing unlocked packages: release_unlocked_beta and release_unlocked_production (#2768)
+ 
+* We added documentation for using CumulusCI to build managed 2GP packages, 
+  unlocked packages, and extending NPSP and EDA with 2GP packages. (#2768)
+
+* Contributions to CCI now require verification by isort (#2770)
+ 
+* CumulusCI now supports deploying unmanaged dependencies in SFDX source format. (#2735)
+
+* The `create_package_version` task now handles dependencies that use a `zip_url`. (#2735)
+
+* Updates to Github Actions configuration documentation. Thanks @Julian88Tex (#2773)
+
+* CumulusCI now automatically recognizes services and orgs configured via environment variables. See the `docs <https://github.com/SFDO-Tooling/CumulusCI/blob/481ab27446edbfdb038ea62834a6f276dc53bf3d/docs/headless.rst#L9>`_ for more details. (#2676 and #2776)
+
+* We've updated the Push Upgrade tasks (push_list, push_sandbox, etc) task option `start_time` to accept ISO-8601 formatted datetimes. (#2769)
+
+* CumulusCI now automatically recognizes enhanced domains in sandboxes when 
+  establishing a connection via JWT. (#2753)
+
+Issues Closed
+
+* Fixed an issue where scratch orgs failed to be deleted in CI environments.
+
+* Fixed an issue where deleting an org failed to mark the org as deleted on CumulusCI's keychain.
+
+* Fixed an issue where CumulusCI would fail on Linux distributions that were incompatible with the `keyring` package. 
```

Comment:
As I said above, put this in Critical Changes and point out that it should no longer be necessary to explicitly set CUMULUSCI_KEY (at least in a CI context where there aren't existing encrypted files to be read using that key)

Reply:
If CUMULUSCI_KEY isn't necessary anymore, why do we still issue a warning? I don't think we can publicize a "critical change" that people should strip it out if it will generate a confusing warning?

Reply:
The warning is trying to say that we're able to store configuration but it won't be encrypted. I agree we should make it less scary but it's too late for this release. I'm okay with holding off on mentioning that CUMULUSCI_KEY is optional until we do that.

Reply:
In particular we should make sure that we aren't showing a warning that suggests installing keyring.alt on Linux, because I don't think that's actually what we want people to do.